nilearn/datasets/func.py

```diff
@@ -1831,10 +1831,10 @@ def load_nki(
          - ``"sphere"``
          - ``"flat"``
 
-    n_subjects : :obj:`int`, default=10
+    n_subjects : :obj:`int`, default=11
```

Comment:
```suggestion
    n_subjects : :obj:`int`, default=1
```
NOT 11!!! haha

Reply:
how say "I am tired" without saying "I am tired"

Reply:
🥲